### DISCUSSION TITLE
Move runtime inputs to advanced options of docker scripts

### DISCUSSION THREAD

**Discussion by MahmoudAlyuDeen** [tteck/Proxmox#3759] [Ideas]
A picture is worth 1000 words:

<img width="685" alt="Screenshot 2024-09-19 at 10 05 20 PM" src="https://github.com/user-attachments/assets/90fdc0c6-07fd-40f8-8b74-23192bf82fdc">

**Comment by tteck** [comment]
It needs to be an option for default installs also. The current configuration works fine.
